config/database.yml

```diff
@@ -11,15 +11,15 @@ default: &default
 
 development:
   <<: *default
-  database: techire_development
+  database: techhire_development
 
 # Warning: The database defined as "test" will be erased and
 # re-generated from your development database when you run "rake".
 # Do not set this db to the same as development or production.
 test:
   <<: *default
-  database: techire_test
+  database: techhire_test
 
 production:
   <<: *default
-  database: techire_production
+  database: techhire_production
```

Comment:
added in the second "h" in "techhire". All will need to re-create their database